api/src/main/resources/emails/egress_remediation_email/content.html

```diff
@@ -0,0 +1,56 @@
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+  <style>
+    body {
+      background-color: #F1F1F1;
+      padding-left: 1rem;
+      padding-top: 1rem;
+      width: 600px;
+    }
+    div {
+      color: #302C71;
+      font-family: 'Montserrat', sans-serif;
+      font-weight: 300;
+      font-size: 16px;
+      line-height: 24px;
+      margin-top: 2rem;
+      margin-bottom: 1rem;
+    }
+
+  </style>
+</head>
+<body>
+<img src="${HEADER_IMG}">
+<div>Sir or Ma'am,</div>
```

Comment:
👎 non-inclusive language.

Our other emails have "Hello firstname lastname" or no greeting at all.  If we need more formality for this one, maybe... Dear Researcher ?

Reply:
@jmthibault79 thanks, noticed this but wasn't sure of a formal alternative. I'll propose "no greeting" to the security team and see what they say.

@yonghaoy I didn't write this, this is copied from their current template. I edited some of the text below for accuracy

Reply:
Hey prefer "Dear researcher," - updated this accordingly